src/components/logical-expression-builder/leb-group/leb-group.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
Use xdescribe instead of comments - like this, the test is still visible in the test list and shows as disabled